com.unity.render-pipelines.universal/Runtime/Passes/PostProcessPass.cs

```diff
@@ -742,54 +693,34 @@ void DoSubpixelMorphologicalAntialiasing(ref CameraData cameraData, CommandBuffe
             }
 
             // Intermediate targets
-            RenderTargetIdentifier stencil; // We would only need stencil, no depth. But Unity doesn't support that.
+            RTHandle stencil; // We would only need stencil, no depth. But Unity doesn't support that.
             DepthBits tempDepthBits;
-            if (m_Depth == k_CameraTarget || m_Descriptor.msaaSamples > 1)
+            if (m_Depth.nameID == BuiltinRenderTextureType.CameraTarget || m_Descriptor.msaaSamples > 1)
             {
                 // In case m_Depth is CameraTarget it may refer to the backbuffer and we can't use that as an attachment on all platforms
-                stencil = ShaderConstants._EdgeTexture;
-                tempDepthBits = DepthBits.Depth24;
+                RenderingUtils.ReAllocateIfNeeded(ref m_EdgeStencilTexture, GetCompatibleDescriptor(m_Descriptor.width, m_Descriptor.height, GraphicsFormat.None, DepthBits.Depth24), FilterMode.Bilinear, TextureWrapMode.Clamp, name: "_EdgeStencilTexture");
+                stencil = m_EdgeStencilTexture;
             }
             else
             {
-                stencil = m_Depth.nameID;
-                tempDepthBits = DepthBits.None;
+                stencil = m_Depth;
             }
-            cmd.GetTemporaryRT(ShaderConstants._EdgeTexture, GetCompatibleDescriptor(m_Descriptor.width, m_Descriptor.height, m_SMAAEdgeFormat, tempDepthBits), FilterMode.Bilinear);
-            cmd.GetTemporaryRT(ShaderConstants._BlendTexture, GetCompatibleDescriptor(m_Descriptor.width, m_Descriptor.height, GraphicsFormat.R8G8B8A8_UNorm), FilterMode.Point);
-
-            // Prepare for manual blit
-            cmd.SetViewProjectionMatrices(Matrix4x4.identity, Matrix4x4.identity);
-            cmd.SetViewport(pixelRect);
+            RenderingUtils.ReAllocateIfNeeded(ref m_EdgeColorTexture, GetCompatibleDescriptor(m_Descriptor.width, m_Descriptor.height, m_SMAAEdgeFormat), FilterMode.Bilinear, TextureWrapMode.Clamp, name: "_EdgeColorTexture");
+            RenderingUtils.ReAllocateIfNeeded(ref m_BlendTexture, GetCompatibleDescriptor(m_Descriptor.width, m_Descriptor.height, GraphicsFormat.R8G8B8A8_UNorm), FilterMode.Point, TextureWrapMode.Clamp, name: "_BlendTexture");
 
             // Pass 1: Edge detection
-            cmd.SetRenderTarget(new RenderTargetIdentifier(ShaderConstants._EdgeTexture, 0, CubemapFace.Unknown, -1),
-                RenderBufferLoadAction.DontCare, RenderBufferStoreAction.Store, stencil,
-                RenderBufferLoadAction.DontCare, RenderBufferStoreAction.Store);
-            cmd.ClearRenderTarget(RTClearFlags.ColorStencil, Color.clear, 1.0f, 0);
-            cmd.SetGlobalTexture(ShaderConstants._ColorTexture, source);
-            DrawFullscreenMesh(cmd, material, 0);
+            RenderingUtils.Blit(cmd, source, pixelRect,
```

Comment:
Can we merge this to the Core.Blitter?
This introduces another Blit again.
Ideally we wouldn't need it, but of course it's understandable if merge is difficult.
What is different here?

Reply:
There's a custom viewport (`pixelRect`) so `Blitter.BlitCameraTexture` cannot be used.
However, this `RenderingUtils.Blit` uses `Blitter.BlitTexture` now after manually setting the viewport.